test/map_constraints_test.rb

```diff
@@ -21,15 +21,14 @@ def self.permits(value, options = {})
   permits({:id => '1'}, :as => {:id => 1})
   permits(:name => 'Mick')
 
-  rejects(:id => 1, :name => 123)
-  rejects(:id => 'Mick', :name => 'Mick')
+  rejects({:id => 1, :name => 123}, :key => :name)
```

Comment:
before it said the hash was invalid, but actually it is just the 'name' key that is invalid, is that better or worse ?